### PR TITLE
Add docs on secrets

### LINKED ISSUE #1369: Docs on Secrets
Add some documentation on how the secrets feature works.

### PR DESCRIPTION
**Pull request by SaraVieira**
closes #1369

### CODE REVIEW

File: packages/homepage/content/docs/5-secrets.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+---
+
+## What are secrets?
+
+Secrets are also known as enviroment variables are used to hide sensitive information in your application that you don't want the world to see.
+The most common example for the usage is with API keys.
+
+These secrets will not be transferred between forks and **can only be used in container sandboxes**
+
+## How add secrets in CodeSandbox
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
## Adding secrets in CodeSandbox
```

File: packages/homepage/content/docs/5-secrets.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  .createServer(function(req, res) {
+    res.write('This your maps key' + key);
+    res.end(); //end the response
+  })
+  .listen(8080);
+```
+
+The sandbox used is [https://codesandbox.io/s/2v6xq474kj](https://codesandbox.io/s/2v6xq474kj)
+
+## How edit secrets in CodeSandbox
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
## Editing secrets in CodeSandbox
```

File: packages/homepage/content/docs/5-secrets.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+authors: ["CompuIves"]
+description: "CodeSandbox allows you to store secrets so you don't share your keys"
+---
+
+## What are secrets?
+
+Secrets are also known as enviroment variables are used to hide sensitive information in your application that you don't want the world to see.
+The most common example for the usage is with API keys.
+
+These secrets will not be transferred between forks and **can only be used in container sandboxes**
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
Secrets values will not be transferred between forks and **can only be used in container sandboxes**.
```

File: packages/homepage/content/docs/5-secrets.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
+title: "Secrets"
+authors: ["CompuIves"]
+description: "CodeSandbox allows you to store secrets so you don't share your keys"
+---
+
+## What are secrets?
+
+Secrets are also known as enviroment variables are used to hide sensitive information in your application that you don't want the world to see.
+The most common example for the usage is with API keys.
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
The most common use case for using secrets is with API keys.
```

**Comment by SaraVieira** [comment]
bless you bogdan

File: packages/homepage/content/docs/5-secrets.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+title: "Secrets"
+authors: ["CompuIves"]
+description: "CodeSandbox allows you to store secrets so you don't share your keys"
+---
+
+## What are secrets?
+
+Secrets are also known as enviroment variables are used to hide sensitive information in your application that you don't want the world to see.
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
Secrets are used to hide sensitive information in your application that you don't want the world to see, and are implemented in CodeSandbox using environment variables.
```

File: packages/homepage/content/docs/5-secrets.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+The most common example for the usage is with API keys.
+
+These secrets will not be transferred between forks and **can only be used in container sandboxes**
+
+## How add secrets in CodeSandbox
+
+You can control the secrets in your server control panel and before adding any it should look like this:
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
You can control the secrets in your Server Control Panel, before adding any it should look like this:
```

File: packages/homepage/content/docs/5-secrets.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+![No Secrets](./images/secrets-1.png)
+
+Let's say you wanna add a google maps api key. You can do it like so:
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
Let's say you want to add a Google Maps API key. You can do it like so:
```

File: packages/homepage/content/docs/5-secrets.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+![Map Secrets](./images/secrets-2.png)
+
+It's always a good practice to name your secrets all in uppercase.
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
Pro tip: It's always a good practice to name your secrets all in uppercase.
```

File: packages/homepage/content/docs/5-secrets.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+After clicking add secret you will a list of all your secrets above the form.
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
After clicking the "Save Secret" button the secret is added, the sandbox is restarted, and you can see the list of all your secrets above the form.
```

File: packages/homepage/content/docs/5-secrets.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+![Secrets](./images/secrets-3.png)
+
+To use them in your code you use `process.env` and the name of your secret so in the case I call `process.env.GOOGLE_MAPS_API_KEY` and it will return me the value.
```
Comment (lbogdan) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
As said earlier, secrets are environment variables, meaning they are defined on `process.env`. In the example above, we can read the API key from `process.env.GOOGLE_MAPS_API_KEY`.
```

File: packages/homepage/content/docs/5-secrets.md (hunk 5 above)
Comment (CompuIves) [reply packages/homepage/content/docs/5-secrets.md]: Great suggestion!

File: packages/homepage/content/docs/5-secrets.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+## Adding secrets in CodeSandbox
+
+You can control the secrets in your server control panel and before adding any it should look like this:
+
+![No Secrets](./images/secrets-1.png)
+
+Let's say you wanna add a google maps api key. You can do it like so:
+
+![Map Secrets](./images/secrets-2.png)
```
Comment (CompuIves) [inline packages/homepage/content/docs/5-secrets.md]: Nice that you have images!

**Comment by CompuIves** [comment]
This is really great!!

File: packages/homepage/content/docs/5-secrets.md [hunk 12]
```diff
@@ -0,0 +1,61 @@
+---
+title: "Secrets"
+authors: ["CompuIves"]
```
Comment (CompuIves) [inline packages/homepage/content/docs/5-secrets.md]:
```suggestion
authors: ["SaraVieira"]
```

😄

File: packages/homepage/content/docs/5-secrets.md (hunk 5 above)
Comment (SaraVieira) [reply packages/homepage/content/docs/5-secrets.md]: Yes!

File: packages/homepage/content/docs/5-secrets.md (hunk 7 above)
Comment (SaraVieira) [reply packages/homepage/content/docs/5-secrets.md]: Yeah, wanna is not a word, I keep forgetting that

File: packages/homepage/content/docs/5-secrets.md (hunk 11 above)
Comment (SaraVieira) [reply packages/homepage/content/docs/5-secrets.md]: ALL THE SCREENSHOTS

File: packages/homepage/content/docs/5-secrets.md (hunk 8 above)
Comment (SaraVieira) [reply packages/homepage/content/docs/5-secrets.md]: Good!

**Comment by SaraVieira** [comment]
All fixed! :D

**Pull request merged**